yarn/src/main/scala/org/apache/spark/deploy/yarn/ClientArguments.scala

```diff
@@ -38,24 +38,30 @@ private[spark] class ClientArguments(args: Array[String], sparkConf: SparkConf)
   var amMemory: Int = 512 // MB
   var appName: String = "Spark"
   var priority = 0
+  var isClusterMode = false
```

Comment:
Ok but I keep `isClusterMode` public to make sure it is accessable in `ClientBase.scala`.